### PR TITLE
Compare signals

### PR DESCRIPTION
**Pull request by pezholio**
Par example:

newplot 1

(Ignore the font - was a png created using Plotly)

### CODE REVIEW

**Comment by pezholio** [comment]
Now pushed to staging too https://goingunderground-staging.herokuapp.com

**Comment by pezholio** [comment]
This is now up to date with #46 - merge that in, then this can go

File: lib/sir_handel.rb [hunk 1]
```diff
@@ -106,10 +119,17 @@ class App < Sinatra::Base
       to = params.fetch('to', default_dates[:to])
       interval = params.fetch('interval', settings.default_interval)
 
+      params[:signal] = params[:signal].split(',').first
+
+      signal = [
+        params[:signal],
+        params[:compare]
+      ].delete_if { |s| s.nil? }.join(',')
```
Comment (pkqk) [inline lib/sir_handel.rb:127]: You could use `#compact` instead of `#delete_if`?

Comment (pezholio) [inline lib/sir_handel.rb:127]: Isn't that an ActiveSupport thing?

Comment (pkqk) [inline lib/sir_handel.rb:127]: nope, [stdlib](http://ruby-doc.org/core-2.2.0/Array.html#method-i-compact)

**Pull request merged**